drivers/nec/display/display.cr

```diff
@@ -0,0 +1,397 @@
+module Nec; end
+module Nec::Display; end
+
+require "placeos-driver/interface/powerable"
+require "placeos-driver/interface/muteable"
+require "placeos-driver/interface/switchable"
+
+class Nec::Display::All < PlaceOS::Driver
+  include Interface::Powerable
+  include Interface::AudioMuteable
+
+  enum Inputs
+    Vga          = 1
+    Rgbhv        = 2
+    Dvi          = 3
+    Hdmi_set     = 4
```

Comment:
Either `CamelCase` or `SCREAMING_CASE` for enum entries. Doesn't matter which one, just as long as it's consistent. 

```suggestion
    HdmiSet      = 4
```

Reply:
yep looks a lot more pleasing in camel case

Reply:
Oh I think doing this https://github.com/PlaceOS/drivers/pull/50/commits/90fc56dbd60dfba97f0d033885fc9337d3391f36#diff-53e26f291e6267f24d41f88df5c2351fR323-R332

means that functions created from this macro will be like `def videoinput` instead of `def video_input` https://github.com/PlaceOS/drivers/pull/50/commits/90fc56dbd60dfba97f0d033885fc9337d3391f36#diff-53e26f291e6267f24d41f88df5c2351fR343-R348

Is it okay to use underscores in the enum entries for this case then @KimBurgess ?

Reply:
nevermind I should of read your comment below https://github.com/PlaceOS/drivers/pull/50#discussion_r489943454